### PR TITLE
[DO NOT MERGE] 1.2.0 release

### PR DESCRIPTION
**Pull request by yannham**
This PR is intended for review purposes, and shouldn't be merged.

1.2.0 release tasks.

### CODE REVIEW

File: RELEASES.md [hunk 1]
```diff
@@ -1,3 +1,77 @@
...
+* LSP Fix two issues with incomplete input by @jneem in https://github.com/tweag/nickel/pull/1550
+* Fix incomplete check of record row constraints by @yannham in https://github.com/tweag/nickel/pull/1558
+* Fix LSP regression on hovering by @yannham in https://github.com/tweag/nickel/pull/1583
+
+Tooling
+-------
+
+* Full vscode extension by @szlend and @yannham in (https://github.com/tweag/nickel/pull/1405, https://github.com/tweag/nickel/pull/1413, https://github.com/tweag/nickel/pull/1416)
+* Invalidate importers in NLS (plus a couple other import-related issues) by @jneem in https://github.com/tweag/nickel/pull/1426
+* Improve `nickel query` (and `:query` in the REPL) inteface by @yannham in
```
Comment (yannham) [inline RELEASES.md:41]:
```suggestion
* Improve `nickel query` (and `:query` in the REPL) interface by @yannham in
```

File: RELEASES.md [hunk 2]
```diff
@@ -1,3 +1,77 @@
...
+
+Tooling
+-------
+
+* Full vscode extension by @szlend and @yannham in (https://github.com/tweag/nickel/pull/1405, https://github.com/tweag/nickel/pull/1413, https://github.com/tweag/nickel/pull/1416)
+* Invalidate importers in NLS (plus a couple other import-related issues) by @jneem in https://github.com/tweag/nickel/pull/1426
+* Improve `nickel query` (and `:query` in the REPL) interface by @yannham in
+  https://github.com/tweag/nickel/pull/1447
+* Topiary integration (formatting) as `nickel format` by @vkleen in https://github.com/tweag/nickel/pull/1371
+* LSP: various improvement to completion by @jneem (https://github.com/tweag/nickel/pull/1450, https://github.com/tweag/nickel/pull/1473)
```
Comment (yannham) [inline RELEASES.md:44]:
```suggestion
* LSP: various improvements to completion by @jneem (https://github.com/tweag/nickel/pull/1450, https://github.com/tweag/nickel/pull/1473)
```

File: RELEASING.md [hunk 3]
```diff
@@ -108,26 +109,15 @@ Other crates carry the version number of the Nickel language. These are
...
+   Make sure that you set `X.Y.Z-release` as the target branch and have GitHub
+   create the `X.Y.Z` tag on release. Upload `nickel-arm64-docker-image.tar.gz`
+   and `nickel-arm64-linux` as release assets from the last step.
 
-3. Verify that the "Upload release artifacts" GitHub action is getting triggered
-   and completes successfully, uploading a static Nickel binary for x86_64 Linux
-   and a Docker image.
+2. Verify that the "Upload release artifacts" GitHub action is getting triggered
+   and completes successfully, uploading a static Nickel binary and a Docker
+   image for both x86-64 Linux and arm64 MacOS platforms.
```
Comment (vkleen) [inline RELEASING.md:120]:
```suggestion
   image for both x86-64 and arm64 Linux platforms.
```

**Review by jneem — APPROVED** [review]

**Comment by yannham** [comment]
Thanks for the reviews!

**Pull request closed without merging**